templates/sounds/after_download_modal_donation.html

```diff
@@ -3,9 +3,15 @@
         Support Freesound
     </div>
     <div class="modal-body">
-        Thanks for downloading <b>{{ sound_name }}</b>!<br />
-        Please consider supporting us by
-        <a href="{% url 'donate' %}">making a donation</a> if you have not contributed yet.<br />
+        <p>Thanks for downloading <b>{{ sound_name }}</b>!</p>
+        <p>It looks like you have been donwloading a singificant number of sounds recently.
```

Comment:
It looks like you have downloaded a significant number....

check spelling - 2 errors here